_studio/mfx_lib/encode_hw/vp9/include/mfx_vp9_encode_hw_vaapi.h

```diff
@@ -65,45 +65,39 @@ namespace MfxHwVP9Encode
         mfxStatus CreateAuxilliaryDevice(
             VideoCORE* core,
             GUID       guid,
-            VP9MfxVideoParam const & par);
+            VP9MfxVideoParam const & par) override;
 
         virtual
         mfxStatus CreateAccelerationService(
-            VP9MfxVideoParam const & par);
+            VP9MfxVideoParam const & par) override;
 
         virtual
         mfxStatus Reset(
-            VP9MfxVideoParam const & par);
-
-        // empty  for Lin
-        virtual
-        mfxStatus Register(
```

Comment:
maybe we should return error here?

Reply:
@mgonchar This function (I mean, with that signature) is not present in base class and never called. Why should we keep it?

Reply:
ok